common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -209,7 +215,7 @@ function (VideoPlayer, i18n, moment) {
         }
     }
 
-    function _loadYoutubeApi(state) {
+    function _waitForYoutubeApi(state) {
         console.log('[Video info]: YouTube API is not loaded. Will try to load...');
```

Comment:
yes, first we show this message and when it's available then show the message accordingly.